src/mdns/mdns_mdnssd.cpp

```diff
@@ -249,6 +249,7 @@ void PublisherMDnsSd::Stop(void)
     if (mHostsRef != nullptr)
     {
         DNSServiceRefDeallocate(mHostsRef);
+        otbrLogInfo("Deallocated DNSServiceRef for hosts: %p", mHostsRef);
```

Comment:
```suggestion
        otbrLogDebug("Deallocated DNSServiceRef for hosts: %p", mHostsRef);
```
nit: maybe use debug ?